joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/astcreation/AstForTypesCreator.scala

```diff
@@ -62,82 +52,42 @@ trait AstForTypesCreator { this: AstCreator =>
   }
 
   protected def astForNamespaceAlias(namespaceAlias: ICPPASTNamespaceAlias): Ast = {
-    val linenumber   = line(namespaceAlias)
-    val columnnumber = column(namespaceAlias)
-    val filename     = fileName(namespaceAlias)
-
     val name     = ASTStringUtil.getSimpleName(namespaceAlias.getAlias)
     val fullname = fullName(namespaceAlias)
 
     if (!isQualifiedName(name)) {
       usingDeclarationMappings.put(name, fullname)
     }
 
-    val code = "namespace " + name + " = " + fullname
-    val cpgNamespace = NewNamespaceBlock()
-      .code(code)
-      .lineNumber(linenumber)
-      .columnNumber(columnnumber)
-      .filename(filename)
-      .name(name)
-      .fullName(fullname)
-
+    val code         = "namespace " + name + " = " + fullname
```

Comment:
better use string interpolation - more readable and safer
there's a few other places in this PR as well...
```suggestion
    val code         = s"namespace $name = $fullname"
```

Reply:
looks like that's true for the c2cpg codebase - might be worth fixing it - intellij can do that semi-automatically for you...